assets/src/components/supervisor-block-header.tsx

```diff
@@ -1,50 +1,59 @@
-import { IoLockClosedSharp } from 'react-icons/io5';
-import { TfiReload } from 'react-icons/tfi';
-import { RiFileShredLine, RiPlayFill, RiStopFill } from 'react-icons/ri';
-import { ActButton, Type } from './act-button';
-import { AddProgram } from './add-program';
-import { useStore } from '~/main/context-provider';
+import {IoLockClosedSharp} from 'react-icons/io5';
+import {TfiReload} from 'react-icons/tfi';
+import {RiFileShredLine, RiPlayFill, RiStopFill} from 'react-icons/ri';
+import {ActButton, Type} from './act-button';
+import {AddProgram} from './add-program';
+import {useStore} from '~/main/context-provider';
+import {useNameLen} from "~/shared/lib/utils";
 
-export const SupervisorBlockHeader = ({ item }: { item: ApiSupervisor }) => {
-  const { landingStore } = useStore();
+export const SupervisorBlockHeader = ({item}: { item: ApiSupervisor }) => {
+  const {landingStore} = useStore();
 
   const buttons = (
-    <div className='flex items-center space-x-1'>
-      <AddProgram />
-      <ActButton type={Type.Orange} onClick={() => landingStore.clearAllProcessLog(item.server.name)}>
-        <RiFileShredLine />
-      </ActButton>
-      <ActButton type={Type.Green} onClick={() => landingStore.startAll(item.server.name)}>
-        <RiPlayFill />
-      </ActButton>
-      <ActButton type={Type.Red} onClick={() => landingStore.stopAll(item.server.name)}>
-        <RiStopFill />
-      </ActButton>
-      <ActButton type={Type.Blue} onClick={() => landingStore.restartAll(item.server.name)}>
-        <TfiReload />
-      </ActButton>
+    <div className='flex justify-end'>
+      <div className='flex items-center space-x-1'>
+        <AddProgram/>
+        <ActButton type={Type.Orange} onClick={() => landingStore.clearAllProcessLog(item.server.name)}>
+          <RiFileShredLine/>
```

Comment:
Use prettier for formatting

Reply:
fix fb48f6d0236a6101a793588bddb6101c71610c78

Reply:
yep